src/management-system-v2/app/(dashboard)/[environmentId]/executions/deployments-modal.tsx

```diff
@@ -141,35 +138,15 @@ const DeploymentsModal = ({
 
   const openFolder = async (id: string) => {
     const folder = await getFolder(id);
-
     if ('error' in folder) {
       throw new Error('Failed to fetch folder');
     }
 
-    const folderChildren = await getFolderChildren(folder.id);
-    if ('error' in folderChildren) {
+    const folderContents = await getFolderContents(folder.id, environment.spaceId);
```

Comment:
As per the definition for `getFolderContents`, the first parameter should be the spaceId and the second parameter the folderId. Otherwise this throws an error.